regulations/static/regulations/js/source/views/main/reg-view.js

```diff
@@ -179,6 +178,33 @@ var RegView = ChildView.extend({
         this.loadImages();
     },
 
+    openDef: function(defId, term, $link) {
+        // if its the same definition, diff term link
+        if ($('.open-definition').attr('id') === defId) {
+            this.toggleDefinition($link);
```

Comment:
I think this is a valid concern. My goal is to get the code tested with as little refactoring as possible so that we can safely refactor from there.